src/utilities/routeRegex.ts

```diff
@@ -1,7 +1,22 @@
-export function generateRouteRegexPattern(value: string): RegExp {
+import { Resolved, Route } from '@/types'
+
+export function generateRoutePathRegexPattern(route: Resolved<Route>): RegExp {
```

Comment:
the path part is just renamed, this file is now also responsible for generating the `RegExp[]` for a routes query